### PR TITLE
Feature/my dash first change

### PR DESCRIPTION
**Pull request by waffles95**
I corrected the output so that it only says the day of the week. I still am working on how to parse the mm/dd/yyyy format.

### CODE REVIEW

**Comment by waffles95** [comment]
I'm not sure how to delete the Date Finder Code pull request.... Also I had trouble getting my DateUtilsTest to work with this new update.

**Comment by grembinni** [comment]
https://www.mkyong.com/java8/java-8-how-to-convert-string-to-localdate/

example 3

**Comment by waffles95** [comment]
Fixed it so it takes the dd/mm/yyyy format and the utility tester runs positive for it as well. Probably won't be able to work on the next lesson till tomorrow.

File: src/main/java/com/world/hello/Main.java [hunk 1]
```diff
@@ -1,17 +1,23 @@
...
+
 import com.world.hello.lesson2.VowelUtils;
+import com.world.hello.lesson3.DateUtils;
 
 public class Main {
 
-	public static void main(String[] args) {
+    public static void main(String[] args) throws ParseException {
+
+	String word = "Please type a word: ";
```
Comment (grembinni) [inline src/main/java/com/world/hello/Main.java]: check your on save rules, this formating is off.

**Pull request merged**